errors/large-page-data.md

```diff
@@ -14,6 +14,9 @@ To inspect the props passed to your page, you can inspect the below element's co
 document.getElementById("__NEXT_DATA__").text
 ```
 
+Hint: if you have a large navigatio with a lot of `<Link/>` components, check if the prerender prop is set. When it is set to true, `getStaticProps` fetches all the data for all pages and puts it into `__NEXT_DATA__`.
```

Comment:
This doesn't make sense to me, prefetching does not affect the cause of this warning which is returning a large amount of data form getStaticProps/getServerSideProps/getInitialProps. `prefetch` does not cause data for other pages to be part of `__NEXT_DATA__`

Reply:
@timneutkens yes you are correct, the bug was something else. This PR should be closed